addons/payment/models/payment_transaction.py

```diff
@@ -838,8 +840,8 @@ def _get_post_processing_values(self):
             'landing_route': self.landing_route,
         }
         _logger.debug(
-            "post-processing values for acquirer with id %s:\n%s",
-            self.acquirer_id.id, pprint.pformat(post_processing_values)
+            "post-processing values of transaction with reference %s for acquirer with id %s:\n%s",
+            self.reference, self.acquirer_id.id, pprint.pformat(post_processing_values)
```

Comment:
Note in case you'd get too carried away when applying the comments above: this one is fine. Even if the params are not named, it's clear enough and convenient that it fits on only two lines.